irohad/main/iroha_conf_loader.cpp

```diff
@@ -336,11 +337,24 @@ JsonDeserializerImpl::getVal<std::unique_ptr<shared_model::interface::Peer>>(
   getValByKey(path, address, obj, config_members::Address);
   std::string public_key_str;
   getValByKey(path, public_key_str, obj, config_members::PublicKey);
+  boost::optional<std::string> tls_certificate_path =
+      getOptValByKey<std::string>(
+          path, obj, config_members::TlsCertificatePath);
+
+  boost::optional<std::string> tls_certificate_str;
+  if (tls_certificate_path) {
+    iroha::readFile(*tls_certificate_path)
+        .match([&tls_certificate_str](
+                   const auto &v) { tls_certificate_str = v.value; },
+               [this](const auto &e) { assert_fatal(false, e.error); });
```

Comment:
please add the config member `path` to the exception message, like it is done in other places